packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -0,0 +1,436 @@
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th width="size20">Author</Th>
+      <Th>Quote</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr verticalAlignment="top">
+      <Td>Chinua Achebe</Td>
+      <Td>The impatient idealist says: ‘Give me a place to stand and I shall move the earth.’ But such a place does not exist. We all have to stand on the earth itself and go with her at her pace.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>James Baldwin</Td>
+      <Td>Anyone who has ever struggled with poverty knows how extremely expensive it is to be poor.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>Dorothy West</Td>
+      <Td>To know how much there is to know is the beginning of learning to live.</Td>
+    </Tr>
+    <Tr verticalAlignment="top">
+      <Td>W.E.B. Du Bois</Td>
+      <Td>Herein lies the tragedy of the age: not that men are poor—all men know something of poverty; not that men are wicked—who is good? Not that men are ignorant—what is truth? Nay, but that men know so little of men.</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+## Anatomy
+
+### Table
+
+| Property | Default token | Modifiable? |
+| -------- | ------------- | ----------- |
+| color    | color-text    | No          |
+| width    | 100%          | No          |
+
+### TBody
+
+| Property         | Default token         | Modifiable? |
+| ---------------- | --------------------- | ----------- |
+| background-color | color-background-body | No          |
+| border-width     | border-width-10       | No          |
+| border-color     | color-border-lighter  | No          |
+
+### TFoot
+
+| Property     | Default token        | Modifiable? |
+| ------------ | -------------------- | ----------- |
+| border-width | border-width-10      | No          |
+| border-color | color-border-lighter | No          |
+
+### Tr
+
+| Property             | Default token                            | Modifiable? |
+| -------------------- | ---------------------------------------- | ----------- |
+| background-color     | Even rows - color-background-row-striped | No          |
+| border-bottom-width  | border-width-10                          | No          |
+| border-bottom- color | color-border-lighter                     | No          |
+
+### Th
+
+| Property    | Default token                                                            | Modifiable? |
+| ----------- | ------------------------------------------------------------------------ | ----------- |
+| font-size   | font-size-30                                                             | No          |
+| line-height | line-height-30                                                           | No          |
+| font-weight | font-weight-bold                                                         | No          |
+| padding     | space-40, First - padding-left: space-50, Last - padding-right: space-50 | No          |
```

Comment:
I'm having trouble understanding what "first" and "last" are referring to here.

Reply:
Same for the Td table.

Reply:
I've updated this to read a little better. Let me know if that works.

Reply:
Much more clear!